packages/features/ee/organizations/components/AddNewTeamsForm.tsx

```diff
@@ -43,10 +43,10 @@ export const AddNewTeamsForm = () => {
       if (data.duplicatedSlugs.length) {
         showToast(t("duplicated_slugs_warning", { slugs: data.duplicatedSlugs.join(", ") }), "warning");
         setTimeout(() => {
-          router.push(`/getting-started`);
+          router.push(`/event-types`);
         }, 3000);
       } else {
-        router.push(`/getting-started`);
+        router.push(`/event-types`);
```

Comment:
Same as the other place i mentioned 

Do we need this changed? I think this will cause a flash of the wrong content as you need to onboard first

Reply:
There is logic there to skip redirection to user onboarding for orgs as we are going to show an alert for it. I consulted with @Jaibles and he preferred not to have the user fill another wizard after org creation.